app/src/main/java/com/example/moviews/MainActivity.kt

```diff
@@ -2,12 +2,48 @@ package com.example.moviews
 
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import com.example.moviews.screen.favorite.FavoriteFragment
+import com.example.moviews.screen.home.HomeFragment
+import com.example.moviews.screen.information.InformationFragment
+import com.example.moviews.screen.search.SearchFragment
+import kotlinx.android.synthetic.main.activity_main.*
 
 class MainActivity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+        switchFragment(HomeFragment())
+        eventClickBottomNavigationView()
+    }
 
+    private fun eventClickBottomNavigationView() {
+        bottomNavigation.setOnNavigationItemSelectedListener { item ->
+            when (item.itemId) {
+                R.id.navigationHome -> {
+                    switchFragment(HomeFragment())
+                }
+                R.id.navigationSearch -> {
+                    switchFragment(SearchFragment())
+                }
+                R.id.navigationFavorites -> {
+                    switchFragment(FavoriteFragment())
+                }
+                R.id.navigationInfomation -> {
+                    switchFragment(InformationFragment())
+                }
+            }
+            true
+        }
     }
 }
+
+fun MainActivity.switchFragment(fragment: Fragment) {
```

Comment:
Tạo 1 file Activity+.kt 
viết hàm này vào trong file 
```suggestion
fun FragmentActivity.switchFragment(fragment: Fragment) {
```